integration/deployment/prebuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+mkdir -p .cache
+if [ ! -f .cache/kubectl ];then
```

Comment:
this might be a tough ask, but... can't we simply use either a Python, or a Golang Kubernetes client in a programmable manner, rather than relying on the presence of the kubectl binary?
I think you only use it create the CRDs, and the 2 DanmNet manifests? that could be like 50 lines in either language, but minus one dependency

Reply:
kubectl is not required anymore, will be removed.